### DISCUSSION TITLE
How to allow searchable() in combination with options() in form?

### DISCUSSION THREAD

**Discussion by bartclaeys** [filamentphp/filament#4852] [Help] [answered]
I can successfully control what options appear in my form select as follows:

```
BelongsToSelect::make('app_id')
    ->options(Books::orderBy('created_at', 'DESC')->limit(5)->pluck('title'))
    ->searchable()
```
But by doing so `searchable()` stopped working :(

How can I show the last 5 books created AND make sure search in the select field keeps on working?

**Comment by danharrin** [comment, marked as answer]
Just use a normal `Select`. BelongsToSelect is deprecated.

Also, `->pluck('title', 'id')`
